backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -71,11 +71,16 @@ def get_date(data, doc_type):
         )
         thesis_date = get_value(data, "thesis_info.date")
         imprint_date = get_value(data, "imprints.date[0]")
+        earliest_date = get_value(data, "earliest_date")
```

Comment:
there is no explicit `earliest_date` in the record. It's computed from all dates in the record, so it's an attribute on `data` (which is an instance of `LiteratureRecord`): https://github.com/inspirehep/inspirehep/blob/3efec10df15ee432f61aa13d57269568f29a9758/backend/inspirehep/records/api/literature.py#L77